src/api/index.ts

```diff
@@ -7,48 +7,31 @@
 
 import axios from 'axios'
 
-let api: any
 let baseUrl: string = process.env.API_ENDPOINT
 	? process.env.API_ENDPOINT
 	: 'https://jsonplaceholder.typicode.com'
 
-const getData = res => res.data
+export const api: any = axios.create({baseURL: baseUrl})
```

Comment:
should export default a function to init api and we init it in index or App tsx

```
const initApi = ({
	token = window.localStorage.getItem('token'),
	baseURL = (api && api.defaults.baseURL) || envBaseUrl,
	axiosOptions = { headers: {} },
} = {}) => {
	isLoggedIn = Boolean(token)
	api = axios.create({
		baseURL,
		...axiosOptions,
		headers: {
			authorization: token ? `Bearer ${token}` : undefined,
			...axiosOptions.headers,
		},
	})
}

export default initApi
```

Reply:
I did come across this but think we should keep it simple until we got authentication

Reply:
Accepted